src/components/cart.js

```diff
@@ -78,10 +79,28 @@ export default class Cart extends Component {
   get lineItemsHtml() {
     return this.lineItemCache.reduce((acc, lineItem) => {
       const data = Object.assign({}, lineItem, this.options.viewData);
+      const fullPrice = data.variant.priceV2.amount * data.quantity;
+      const formattedPrice = formatMoney(fullPrice, this.moneyFormat);
+      const discountAllocations = data.discountAllocations;
+
+      let totalDiscount = 0;
+      const discounts = discountAllocations.reduce((discountArr, discount) => {
+        const targetSelection = discount.discountApplication.targetSelection;
+        if (targetSelection === 'ENTITLED' || targetSelection === 'EXPLICIT') {
```

Comment:
Are these key words to indicate discounts for line items? If so, maybe a helper function or an array to store these keys would be helpful and more explicit.